### PR TITLE
Features/confirmed units #156205423

### PR DESCRIPTION
**Pull request by fddayan**
https://www.pivotaltracker.com/story/show/156205423

Merge https://github.com/Exygy/sf-dahlia-lap/pull/38 before merging this PR

### CODE REVIEW

File: app/javascript/components/supplemental_application/sections/ConfirmedUnits.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+              field="has_ada_priorities_selected.mobility_impaired"
+              label="Mobility Impaired"/>
+            <CheckboxBlock
+              field="has_ada_priorities_selected.vision_impaired"
+              label="Vision Impaired"/>
+            <CheckboxBlock
+              field="has_ada_priorities_selected.hearing_impaired"
+              label="Hearing Impaired"/>
+          </div>
+        </FormGrid.Group>
```
Comment (chrisdolendo) [inline app/javascript/components/supplemental_application/sections/ConfirmedUnits.js:67]: cool namespacing for the formgrid section here

File: app/services/force/client_factory.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+module Force
```
Comment (chrisdolendo) [inline app/services/force/client_factory.rb:1]: :+1:

**Review by chrisdolendo — COMMENTED** [review]
@fddayan the code looks good and I like your re-organization of the salesforce base, but the PT story is lacking review instructions. Please update asap: https://www.pivotaltracker.com/story/show/156205423

I tried to use a url path from a past accepted supp app story but couldn't get to the right place. So please ping me when the review instructions are up so I can approve :)

**Review by chrisdolendo — APPROVED** [review]
:+1: Feature functions acc to acceptance criteria. That said, breadcrumb urls need to be updated, I will capture this as a separate story.

File: config/salesforce/fields.yml [hunk 3]
```diff
@@ -1,290 +1,291 @@
-field_update_comment:
-  index_fields:
-    Application__c:
-    Processing_Comment__c:
-    Processing_Date_Updated__c:
-    Processing_Status__c:
+# field_update_comment:
```
Comment (callaghanc) [inline config/salesforce/fields.yml]: @fddayan why is this file commented out?

Comment (fddayan) [reply config/salesforce/fields.yml]: I splitted the files and moved each individual section to `saleforce/fields/` folder. Actually, I have to remove this file

**Pull request merged**